Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Blender/AddOns/SceneExporter/o3de_utils.py

```diff
@@ -24,22 +26,69 @@ def LookatEngineManifest():
             o3de_projects = data['projects']
             # Send if o3de is not installed
             engine_is_installed = True
+            o3de_projects = extend_project_list(o3de_projects)
             return o3de_projects, engine_is_installed
     except:
         o3de_projects = ['']
         engine_is_installed = False
         return o3de_projects, engine_is_installed
 
-def build_projects_list():
+def load_saved_projects():
+    """!

```

Comment:
I'm not sure what the convention being used is with doc string quote `"""!` with the exclamation point

Reply:
We use doxygen to generate documentation. This is in line with the doxygen style guide.